src/server/schemas/User.js

```diff
@@ -0,0 +1,60 @@
+const mongoose = require("mongoose");
+
+mongoose.set("useCreateIndex", true);
+mongoose.set("useNewUrlParser", true);
+mongoose.set("useUnifiedTopology", true);
+const uniqueValidator = require("mongoose-unique-validator");
+const bcrypt = require("bcrypt");
+
+const SALT_WORK_FACTOR = 10;
+
+const UserSchema = new mongoose.Schema({
+  email: {
+    type: String,
+    required: true,
+    unique: true,
+    index: { unique: true },
+  },
+  password: String,
+  decks: [
+    {
+      name: String,
+      cards: [String],
+    },
+  ],
+  friends: [String],
```

Comment:
I think we're gonna want users to have unique IDs (that aren't just their emails), and we're gonna want to save friends using IDs rather than emails -- cuz people like to change the emails they have on file